.cirrus.yml

```diff
@@ -46,3 +43,11 @@ task:
     DEPLOY_TOKEN: ENCRYPTED[fbf2f9b7dc97cc3d960126b1540f19c36b96768c46b8eb99389e02b8c54080418d1fb90094dc2e3a36767df34acb5d5d]
   install_script: pip install --upgrade pymdown-extensions
   deploy_script: ./.ci/deploy.sh
+
+task:
+  name: Danger PR
+  env:
+    DANGER_GITHUB_API_TOKEN: ENCRYPTED[fbf2f9b7dc97cc3d960126b1540f19c36b96768c46b8eb99389e02b8c54080418d1fb90094dc2e3a36767df34acb5d5d]
+  only_if: $CIRRUS_PR != ''
+  install_script: yarn global add danger
```

Comment:
I think you can use `registry.gitlab.com/gitlab-org/gitlab-build-images:danger` image for doing here. 🤔

Reply:
No, sadly we can't, that is Ruby danger in the container (not Node danger), and it gets updated very often so it seems safer to keep it on latest.